Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM openjdk:22-slim
+
+RUN mkdir /app
+
+RUN apt-get update && \
+    apt-get install -y ffmpeg && \
+    rm -rf /var/lib/apt/lists/*
+
+COPY dlnatoad-1-SNAPSHOT-jar-with-dependencies.jar /app/dlnatoad.jar
+
+WORKDIR /app
+
+EXPOSE 8080
```

Comment:
maybe add `--port 8080` to the args below?
(if --port is not specified it will default to 8192)

though the user will need to edit this file anyway to specify which media files to serve... and likely other flags they want.